src/datepicker/datepicker.component.ts

```diff
@@ -271,7 +271,7 @@ export class DatePicker implements OnDestroy, OnChanges, AfterViewChecked {
 	 */
 	protected updateClassNames() {
 		if (!this.elementRef) { return; }
-
+		document.querySelector(".flatpickr-calendar").addEventListener("click", (event) => { event.stopPropagation(); });
```

Comment:
Probably want to add the `.open` class to your query selector in case there are two date pickers

Reply:
good point!